PWGCF/DataModel/FemtoDerived.h

```diff
@@ -191,15 +295,15 @@ DECLARE_SOA_TABLE(FDExtParticles, "AOD", "FDEXTPARTICLE",
                   track::DcaXY,
                   track::DcaZ,
                   track::TPCSignal,
-                  femtodreamparticle::TPCNSigmaEl,
                   femtodreamparticle::TPCNSigmaPi,
                   femtodreamparticle::TPCNSigmaKa,
                   femtodreamparticle::TPCNSigmaPr,
-                  femtodreamparticle::TPCNSigmaDe,
-                  femtodreamparticle::TOFNSigmaEl,
                   femtodreamparticle::TOFNSigmaPi,
                   femtodreamparticle::TOFNSigmaKa,
                   femtodreamparticle::TOFNSigmaPr,
+                  femtodreamparticle::TPCNSigmaEl,
```

Comment:
This in the end was not a change, hence can you please revert to the original order, to keep the list of changes clean.
Thanks!

Reply:
@sartozza done